openff/bespokefit/executor/services/qcgenerator/cache.py

```diff
@@ -50,6 +54,94 @@ def _canonicalize_task(task: _T) -> _T:
     return task
 
 
+def _sha256_hash(contents: str) -> str:
```

Comment:
As we are only using this to hash tasks does it make sense to make it a method of the base `QCGenerationTask`?

Reply:
This may be a good shout, although in the future we may need to hash other types such as internal intermediates depending on how the schemas are changed to for example allow pre-optimization. I'll leave it for now in this PR but feel free to make this change in #167!